apps/io500/readme.md

```diff
@@ -0,0 +1,29 @@
+# IO-500 HPC Storage benchmark
+
+The [IO-500](https://www.vi4io.org/std/io500/start) is a I/O storage benchmark designed to measure and compare different HPC storage solutions. It runs a suite of throughput, IOPS and metadata benchmarks and gives a score to rank the storage solution. It uses modified versions of LLNL IOR/MDtest and a find benchmark.
+
+The IO-500 benchmark needs to be built on HB or HC sku's with CentOS-HPC 7.6 (using mpi/mpich-3.3). See one of the examples for building a Cluster with HB or HC skus and PBS. (e.g. [simple_hpc_pbs](../../examples/simple_hpc_pbs/readme.md))
+
+log-on to the headnode
+```
+azhpc-connect headnode
+```
+
+Copy apps dir to the headnode
+```
+azhpc-scp -r $azhpc_dir/apps hpcuser@headnode:.
+```
+
+Build the IO-500 benchmark suite from the build script.  This will install IO-500 into /apps/io-500-dev and create a modulefile:
+```
+    build_io500.sh
+```
+
+Now submit and run the IO-500 benchmark (e.g on HB):
+```
+     qsub -l select=2:ncpus=60:mpiprocs=15 io500.pbs
+```
+
+> Note: this will run on 2 node and 15 processes per node.
```

Comment:
Please document when the outputs are and how they look like.

Reply:
done.